torch/csrc/Generator.cpp

```diff
@@ -81,7 +81,7 @@ static PyObject * THPGenerator_setState(THPGenerator *self, PyObject *_new_state
   if (!THPVariable_Check(_new_state)) {
     throw TypeError("expected a torch.ByteTensor, but got %s", Py_TYPE(_new_state)->tp_name);
   }
-  auto& tensor = ((THPVariable*)_new_state)->cdata.data();
+  auto& tensor = ((THPVariable*)_new_state)->cdata;
```

Comment:
The original use of `.data()` here is to make sure that the `THByteTensor_setRNGState(generator, (THByteTensor*)tensor.unsafeGetTensorImpl())` call later is using the Variable's underlying `data_`'s TensorImpl instead of the Variable's `Variable::Impl`. Since now there is no `data_` and a Variable stores its tensor data directly in its TensorImpl, and the `THByteTensor_setRNGState(...)` call doesn't care about whether the TensorImpl has autograd metadata, it's okay to remove `.data()` here.